.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended',
+  ],
+  parserOptions: {
+    ecmaVersion: 2021,
+    sourceType: 'module',
+  },
+  plugins: ['prettier', '@typescript-eslint'],
+  rules: {
+    '@typescript-eslint/ban-types': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/explicit-module-boundary-types': [
+      'error',
```

Comment:
Could you tell a little bit more of why you disabled these rules ?

Reply:
To conform with current codebase style and how the code was written. Basically not to unnecessarily bend the code around the someone else's defaults, but make the tool work for you.

Reply:
I think, we should not personalize too much the `eslint` rules, as you said :
> Linting is often very opinionated, so I tried not to introduce too much opinion and keep the codebase with very little change, but to make it more consistent with already "used" style it was written with.

We should keep the default `eslint` rules and try to rewrite what need to be rewrite if the default rules aren't working.